package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc -p tsconfig.json && npm run copy:files",
     "build:watch": "tsc -p tsconfig.json --watch && npm run copy:files",
-    "copy:files": "cp ./package.json dist/ && cp ./src/*.json dist/ && cp ./src/svg-icons-builder/*.json ./dist/svg-icons-builder/",
+    "copy:files": "cp ./src/builders.json ./src/collection.json dist/ && cp ./package.json dist/ && cp ./src/svg-icons-builder/*.json ./dist/svg-icons-builder/",
```

Comment:
I had to make sure that the dummy `package.json` isn't overwriting the real one.

Reply:
I removed the dummy `package.json`, not needed with the structure change.